tests/utils/test_migration_helpers.py

```diff
@@ -187,8 +200,13 @@ def func_non_optional_params_multiple_params_stacked_duplicate(
 
 def test_deprecate_parameter_stacked_inverse():
     # check: raise if used without kwargs
-    with pytest.warns(DeprecationWarning,):
+    with pytest.warns(DeprecationWarning,) as warn_record:
```

Comment:
Is it expected to be a tuple, i.e., comma at the end?
```
DeprecationWarning,
```

Reply:
cleaned up